src/classes/modules/java.base/java/lang/System.java

```diff
@@ -33,7 +34,13 @@ public class System {
 
   static Properties properties;
 
-  public static InputStream in; // new BufferedInputStream(...);  // <2do> not yet
+  public static InputStream in = new InputStream() {
+    public int available() { return 0; }
```

Comment:
Please add a comment here:
// These are stubs that provide an InputStream instance that has no data available.

Reply:
Code now updated with this comment, @cyrille-artho.
Thanks!